collections/ocean-net-primary-production.yaml

```diff
@@ -0,0 +1,80 @@
+Name: BICEP
+Description: |
+  The data comes from the Copernicus Sentinel-5P satellite and uses data from the Copernicus Sentinel-5P satellite and shows the averaged methane concentrations across the globe — using weekly averaged maps. The methane map shown here is measured by the Tropomi instrument on the Sentinel 5 Precursor satellite.
+  The Copernicus Sentinel-5P CH4 measurements were first filtered according to the recommendation in the Product Readme file (only data with a qa_value > 0.50 was used). Then the measurements are mapped on a fixed latitude-longitude grid of 4096 x 8192 pixels. The grid is turned into an EPSG:4326 geotiff file using the appropriate color scale, which is again turned into an EPSG:3857 tile map.
+  Data gaps are visible based on the product quality filtering and the fact that over the sea only measurements for sun-glint situations are being provided.
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: ocean-net-primary-production/README.MD
+Image: ocean-net-primary-production/npp.png
+EDCBrowser: ""
+Resolution: xx x xx Km (across x along track)
+GeographicalCoverage: -180.0, -90.0, 180.0, 90.0
+TemporalAvailability: 1998 - 2018
+BandInformation: pp
```

Comment:
Please follow the [default structure](https://github.com/eurodatacube/public-collections/blob/main/CONTRIBUTING.md#yaml-file-structure).